chainer/iterators/multiprocess_iterator.py

```diff
@@ -29,11 +34,15 @@ class MultiprocessIterator(iterator.Iterator):
             order of indexes.
         n_processes (int): Number of worker processes. The number of CPUs is
             used by default.
+        n_prefetch (int): Number of prefetch batches.
```

Comment:
Is there any reason to use a value other than the default of 1? Of so, please consider to document the benefit or tradeoffs in using larger values.